packages/components/organisms/f-checkout/src/store/checkoutAnalytics.module.js

```diff
@@ -125,6 +120,17 @@ export default {
                     dispatch('trackFormInteraction', { action: 'error', error: mappedError });
                 }
             });
+        },
+
+        /**
+         * Fetches the variant of the Low Value Order experiment from the headers and pushes an analytics event.
+         */
+        trackLowValueOrderExperiment: experimentHeaders => {
+            const lowValueOrderExperimentVariant = experimentHeaders[HEADER_LOW_VALUE_ORDER_EXPERIMENT];
+            const event = experimentService.getLowValueOrderExperimentTracking(lowValueOrderExperimentVariant);
+            if (event) {
+                window.dataLayer.push(event);
```

Comment:
Do we need to check that the dataLayer exists at all?

Reply:
There are other functions above which don't check but happy to add it